example/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary")
+load("@magic_enum//bazel:copts.bzl", "COPTS")
+
+_EXAMPLES = [
+    "enum_flag_example",
+    "example",
+    "example_containers_array",
+    "example_containers_bitset",
+    "example_containers_set",
+    "example_custom_name",
+    "example_switch",
+]
+
+[cc_binary(
+    name = example,
+    srcs = ["{}.cpp".format(example)],
+    deps = ["@magic_enum"],
+    copts = COPTS,
+) for example in _EXAMPLES]
+
+cc_binary(
+    name = "example_nonascii_name",
+    srcs = ["example_nonascii_name.cpp"],
+    deps = ["@magic_enum"],
+    copts = COPTS,
+    target_compatible_with = select({
+        "@bazel_tools//tools/cpp:msvc": ["@platforms//:incompatible"],

```

Comment:
I'm not sure if this is expected to work with MSVC, but for me it failed so I marked it as incompatible.

Reply:
Can we turn `nonascii_name` off by default?

Reply:
I added a `"manual"` tag which means it may only be built explicitly i.e. `bazel run //example:example_nonascii_name`. Does that work for you?

Alternatively I could add a constraint which would require upstream users to declare they support  the nonascii feature in their platform definition. https://bazel.build/docs/configurable-attributes#platforms